### PR TITLE
x-timeplus http header

### PR DESCRIPTION
**Pull request by chenziliang**
PR checklist:
- Did you run ClangFormat ?
- Did you separate headers to a different section in existing community code base ?
- Did you surround `proton: starts/ends` for new code in existing community code base ?

Please write user-readable short description of the changes:

### CODE REVIEW

**Comment by chenziliang** [comment]
@ye11ow @zliang-min ,  fyi, please let us know if there are dependencies regarding the `x-proton-` headers in neutron etc layers

File: src/Server/RestHTTPRequestHandler.cpp [hunk 1]
```diff
@@ -22,160 +22,158 @@ namespace DB
...
+extern const int AUTHENTICATION_FAILED;
+
+extern const int HTTP_LENGTH_REQUIRED;
+extern const int INVALID_SESSION_TIMEOUT;
 }
 
 namespace
 {
-    String base64Decode(const String & encoded)
+String base64Decode(const String & encoded)
```
Comment (chenziliang) [inline src/Server/RestHTTPRequestHandler.cpp:74]: Reformatting ...

**Comment by zliang-min** [comment]
Unless the proton SDK relies on such headers, otherwise, Neutron and other components should not use any such headers.

**Review by ye11ow — APPROVED** [review]
Neutron use basicAuth to talk to Proton rest. That's the only header we use right now on Neutron side.

**Pull request merged**